src/lib/coaler/io/OutputWriter.cpp

```diff
@@ -10,18 +10,22 @@
 #include <spdlog/spdlog.h>
 
 namespace coaler::io {
-
     void OutputWriter::writeSDF(const std::string &file_path, const coaler::multialign::MultiAlignerResult &result) {
+        if (result.inputLigands.size() != result.poseIDsByLigandID.size())
+            spdlog::warn(fmt::format("received less output molecules than there was in input: {}/{}", result.poseIDsByLigandID.size(), result.inputLigands.size()));
+
```

Comment:
Wollen/Können wir angeben wo die verschwinden?

Reply:
1. das wird noch gefixt werden in einem anderen branch. hat mit den starting assemblies zu tun. sobald wird dyn confgen im opt haben fällt das problem weg